ebuild-maintenance/general/text.xml

```diff
@@ -0,0 +1,527 @@
+"A better ebuild workflow with pure git and pkgcheck"</uri> guide if you're not
+too familiar with git.
+</p>
+
+<p>
+Gentoo's
+<uri link="https://qa-reports.gentoo.org/output/gentoo-ci/output.html">CI
+system </uri> runs pkgcheck, so familiarizing yourself with pkgcheck and its
+warnings is encouraged. If you add a new warning to the tree, you will get an
+e-mail from "repomirrorci@g.o" with the warning, and this could most likely
+have been prevented by running <c>pkgcheck scan --commits</c> before pushing
+your changes. A new CI error will prevent metadata being generated to Gentoo's
+sync-repositories, and therefore deny users the latest updates until the CI
+error is fixed.
+</p>
+
+<p>
+By default <c>pkgcheck</c> enables a lot of checks. Some of them are more
+useful than the others. Pkgcheck can be configured via the <c>
+~/.config/pkgcheck/pkgcheck.conf</c> file. For example, a sane configuration
+could look like this:
+</p>
+
+<pre>
+# ~/.config/pkgcheck/pkgcheck.conf
+#
+# -ImlateCheck = PotentialStable, all of these checks are just noise.
+[DEFAULT]
+checks = -ImlateCheck,-RedundantVersionCheck,-UnstableOnlyCheck
+</pre>
+
+<p>
+Refer to the <uri link="https://pkgcore.GitHub.io/pkgcheck/man/pkgcheck.html">
+official documentation</uri> of the flags and edit to your needs.
+
+You can also just enable the checks you wish from the command line, e.g., <c>
+pkgcheck scan -c StableRequestCheck</c>. More on that below.
+</p>
+
+<p>
+Ask a fellow developer if you're unsure of the meaning or how to fix a warning.
+If in doubt about who to ask, head to the #gentoo-dev, #gentoo-qa, or
+#gentoo-dev-help (for non-developers too) IRC channels. You can, and you
+should, of course ask before pushing if something is not fully clear to you.
+</p>
+
+</body>
+
+<subsubsection>
+<title>GitHub pull requests</title>
+<body>
+
+<p>
+You may find yourself in a lucky position where enthusiastic users provide
+contributions to your packages via GitHub pull requests. Using GitHub is not
+mandatory and Bugzilla is still the main channel for reporting enhancements to
+your packages. But consider trying out GitHub, as it's getting more and more
+popular and we're seeing more and more contributions pushed there. Since a
+distribution would be nothing without its users, we shouldn't ignore our
+users in popular platforms. Chances are they end up becoming official
+developers with your aid.
+</p>
+
+<p>
+<b>Joining the Gentoo GitHub organization:</b> To join the Gentoo GitHub
+organization, make sure you've set your <c>gentooGitHubUser</c> attribute in
+<uri link="https://wiki.gentoo.org/wiki/Project:Infrastructure/LDAP_Guide#Optional_custom_attributes">
+dev.gentoo.org's LDAP record</uri>. Join <c>#gentoo-infra</c> on the Libera IRC
+network to ask for a manual re-sync, and you will be added to the Gentoo
+organization in GitHub. You should be automatically added to teams you're part
+of, but you can request to join manually too. Remember to once again update
```

Comment:
You can but AFAIK (I tried before) we should just poke infra to run the sync script instead, or it'll get lost if for some reason it's not recorded properly on the wiki:

```suggestion
of. Remember to once again update
```